plugins/HostWindows.cpp

```diff
@@ -42,7 +40,7 @@ Modules HostWindows::modules() const {
 	MODULEENTRY32 me;
 	me.dwSize = sizeof(me);
 	for (auto ok = Module32First(snapshotHandle, &me); ok; ok = Module32Next(snapshotHandle, &me)) {
-		const auto name = utf16ToUtf8(reinterpret_cast< char16_t * >(me.szModule));
+		const auto name = utf16To8(me.szModule);
```

Comment:
Now that the cast is gone, we shouldn't use `auto` anymore

Reply:
Please note that the type was already "hidden" before. The cast was for the argument to `utf16ToUtf8()`.

Reply:
True - then it is high time to remedy this issue :)

Reply:
Yeah, I did that.